src/videojs.markers.js

```diff
@@ -287,7 +287,12 @@
          if (newMarkerIndex != currentMarkerIndex) {
             // trigger event
             if (newMarkerIndex != -1 && options.onMarkerReached) {
-              options.onMarkerReached(markersList[newMarkerIndex]);
+               var nextMarker = markersList[newMarkerIndex];
+               // do not auto trigger when user jumps over marker(s)
+               var triggerAccuracy = player.duration() * 0.01;
```

Comment:
What's the reason behind setting this based on player duration instead of an absolute number?